app/services/idv/flows/in_person_flow.rb

```diff
@@ -5,7 +5,8 @@ class InPersonFlow < Flow::BaseFlow
 
       STEPS = {
         state_id: Idv::Steps::InPerson::StateIdStep, # info from state id
-        address: Idv::Steps::InPerson::AddressStep,  # entering the address
+        residential_address: Idv::Steps::InPerson::ResidentialAddressStep, # put residential address
```

Comment:
this is a bit nit picky but can we change `put` in the comment to `enter` to match the comment for `address`